workers/unity/Packages/io.improbable.gdk.debug/Templates/UpdateRow.uss

```diff
@@ -0,0 +1,22 @@
+.item {
```

Comment:
Where did the colors in this come from? 

An alternative would be to do something like `rgba(255, 255, 255, x)` where 0 < x < 1 to overlay colors

Reply:
The color was based on the grey lines between the buttons in the header. Something I picked and looked oke in both dark and light schemes